hybrasyl/Objects/Creature.cs

```diff
@@ -854,6 +851,14 @@ public virtual bool UseCastable(Castable castObject, Creature target = null)
                 }
                 if (castObject.Effects?.Damage != null)
                 {
+                    if (tar.DeathPileAllowedLooters.Count != 0 && !(tar.DeathPileAllowedLooters.Contains(Name))) continue;
+                    if (this is User && !tar.DeathPileAllowedLooters.Contains(Name))
+                    {
+                        var theUser = this as User;
+                        if (theUser.Grouped) tar.DeathPileAllowedLooters = theUser.Group.Members.Select(user => user.Name).ToList();
```

Comment:
DeathPileAllowedLooters should not be used for Item drops from mobs. New functionality should be created to handle that.

DeathPileAllowedLooters should be used for user death piles.